tests/yam/agama/auto.pm

```diff
@@ -8,12 +8,25 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 
-use testapi 'assert_screen';
-
+use testapi;
+use utils 'clear_console';
 
 sub run {
     assert_screen('agama-main-page', 120);
-    assert_screen('agama-installing', 60);
+    assert_screen('agama-installing', 120);
+
+    assert_screen('agama-install-finished', 1200);
+
+    $testapi::password = 'linux';
```

Comment:
The password should be nots3cr3t instead of linux if agma install install finished, if yes then we should submit a bug.

Reply:
@hjluo we update the password in base class, there is no need for this anymore.